### ISSUE TITLE
SF iOS Workshop (Mar-2016)

### ISSUE THREAD

**Issue by mrh-is** [mobilebridge/organizing#17]
Let's have a workshop!

**Comment by dvidsilva** [comment]
If you'd like to host in Berkeley, we can have it at a friend's coworking space. The east bay doesn't get so many events, and this one would be great to have.

**Comment by kmolo** [comment]
@dvidsilva we definitely should have some workshops over in the East Bay; it's up to @mrh-is if he wants to organize this one there, or if we should open a separate issue for an East Bay workshop.

A separate issue is that the hard-to-find resource is funding for food (roughly $1000):

https://github.com/mobilebridge/organizing/issues/14#issuecomment-144911808

...though the smaller workshop, the smaller the budget.

Would some entity in Berkeley be able to fund the food?

**Comment by dvidsilva** [comment]
Yep, both good points. I think there are entities on the east bay that will be happy to support, I have hosted other events for my meetup there and there's good support.

**Comment by mrh-is** [comment]
I'm down for Berkeley! Before we decide that, I'd be interested to know if we have any data/inkling about how many potential students would be affected by this change. Do we have students asking for an East Bay workshop, and would all the SF people come over? Any idea on that @kmolo (or someone to defer to)?

**Comment by kmolo** [comment]
@mrh-is definitely has been interest from Berkeley + Oakland folks, plus Silicon Valley too.

---
Also just saw your awesome meetup coming up:

http://www.meetup.com/swift-language/events/228042292

...sadly, can't go as the CodePath University iOS bootcamp meets Tue + Thu evenings.  Will you record this presentation?  (he default installation of Quicktime on Mac + a USB microphone is all that's necessary)?

**Comment by mrh-is** [comment]
Hmm, could be fun. I guess the drawback is that it's harder for Peninsula folks to come, AFAIK? My initial feeling is that centrally located is good for such a small event. If that's not true, then I totally think we should try out an East Bay version!

* * *

Yes, we record all the talks, and they go up after just a couple weeks. 😁 We have a fancy setup, really high production quality. Happy to point you directly to the relevant links when they go live!

**Comment by kmolo** [comment]
@originaldon

**Comment by kmolo** [comment]
@hleetwin

**Comment by kmolo** [comment]
@sidhene

**Comment by sidhene** [comment]
Hello :)
I just tweeted at Michael

**Comment by sidhene** [comment]
I have a few contacts, I will try to find someone willing to host and pay for food in San Francisco.

**Comment by mrh-is** [comment]
Yay! 🎉 Very very happy to have a partner.

I will also reach out to more people! I have some people I should follow up with as well.

Something that I haven't spent a ton of time thinking about is what population we want to target. So far the iOS workshops have targeted women. I love the workshops so far, but I'm also happy to try a different group if anyone has a suggestion!

**Comment by stellasu168** [comment]
I'd like to be a TA for the iOS workshop. Please let me know the details.

Thanks!
Stella

On Mon, Jun 27, 2016 at 7:45 PM, kmolo <notifications@github.com> wrote:

> @sidhene <https://github.com/sidhene>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mobilebridge/organizing/issues/17#issuecomment-228933771>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD2vL84VkF3D0o8VNleZRUFh1uqykYqOks5qQIrYgaJpZM4HALsX>
> .
>



-- 
Stella Su

**Comment by mmesarina** [comment]
Me too

-malena

**Comment by jeffreybergier** [comment]
I'm a-go for teaching or TA'ing or being helpful in any way possible.

**Comment by cbowns** [comment]
I can help with pre-workshop content/planning

**Comment by nevyn** [comment]
something's happening here? I'm pretty sure Lookback would be fine with sponsoring food again :)

**Comment by sidhene** [comment]
@nevyn - Thanks a lot! We have trouble finding someone to sponsor both, so that would actually help a great deal.

@mrh-is I've found several potential SF sponsors: Pivotal Labs, RedFin, and Talener. I'd like to try and make a go of RedFin, since it's entirely objective (DevBootCamp and Talener are all potential sales to attendees) and would cement a relationship with Women Who Code who have ongoing support groups for both iOS and Android as well as a list of potential space sponsors and interest in collaborating with us. 

The easiest space sponsor is DevBootCamp, now that we have food sponsorship from @nevyn we are unblocked and can move forward. I say it's time to pick a date.

**Comment by sidhene** [comment]
BTW, Pivotal now has an official "fill out this form to use our space" thing: https://docs.google.com/forms/d/1iAnYFWkK0AT8CM8uMqSUORZkfE4ggScJAbpswTepSzw/viewform?edit_requested=true

**Comment by kmolo** [comment]
Wondering if someone could update the needed minimum OS X and Xcode for Swift 3.0

https://github.com/mobilebridge/iosbridge/wiki/Student-Prerequisites

**Comment by jeffreybergier** [comment]
@kmolo I updated the pre-requisites for Xcode 8. If anyone can proof read it and make corrections that would be awesome.

I removed the cocoa pods install part. I don't think we've been using cocoa pods the past few workshops and took up the vast majority of the installation instructions.

I'm not opposed to putting it back, if we still think we need it.

**Comment by mmesarina** [comment]
Is there an IOS teaching event on the works? I would love to help as a
Teacher.

Best

-Malena

**Comment by imju** [comment]
Who is going to organize next event?

Keen.io is looking to sponsor an workshop event.  sarahjane@keen.io
Sarah-Jane Morris is a contact person.

Can someone contact her for next event? @kmolo

Thanks,
Imju

**Comment by kmolo** [comment]
@mrh-is @sidhene are organizing this next event.  @nevyn have offered food, so just looking for a space (though if Keen.io is up for sponsoring some food -- say, food at the teacher training -- that would be awesome).

Turning this conversation over to @mrh-is + @sidhene as I'm still organizing for http://codepathuniversity.com (prerequisite algorithm & data structure / technical interviewing mini-course starting in 2 weeks).

@mrh-is @sidhene, how's it going?  luckily you have folks volunteering:

> food -- @nevyn 
> teaching/curriculum -- @mmesarina @jeffreybergier @cbowns @stellasu168 @nevyn 
> leads on venues -- see comments from @imju + @dvidsilva above

**Comment by kmolo** [comment]
I see that @jeffreybergier updated the setup minimum requirements / instructions on 30-Sep-2016:

https://github.com/mobilebridge/iosbridge/wiki/Student-Prerequisites

Are these current?  High enough for Swift 3.0?
